sdks/python/apache_beam/transforms/combiners.py

```diff
@@ -884,17 +888,18 @@ class PerKey(ptransform.PTransform):
 
     @staticmethod
     def add_timestamp(element, timestamp=core.DoFn.TimestampParam):
-      _check_instance_type(KV[T, T], element)
+      _check_instance_type(Tuple[T, T], element)
```

Comment:
I believe you have accidentally changed KV to Tuple here, I think here we do want to check the input has KV[K, V] type.